workmail-message-flow-state-machine/README.md

```diff
@@ -4,43 +4,34 @@ This application enables you to use AWS Step Functions to orchestrate multiple A
 
 ## Setup
 1. Deploy this application via [AWS Serverless Application Repository](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:489970191081:applications~workmail-message-flow-state-machine).
+    1. [Optional] Enter `MachineStateForOutput` to specify state of the Step Function whose output should be returned back to WorkMail. By default, output of the succeeded execution event is used.
```

Comment:
...to specify **the** state of the Step Function...

..By default, **the** output...